CMakeLists.txt

```diff
@@ -10,6 +10,14 @@
 
 cmake_minimum_required(VERSION 2.8.11)
```

Comment:
For Hunter minimum is 3.0:
* https://docs.hunter.sh/en/latest/quick-start/cmake.html